resources/aws/peering.go

```diff
@@ -79,10 +84,19 @@ func (v *VPCPeeringConnection) CreateIfNotExists() (bool, error) {
 }
 
 func (v *VPCPeeringConnection) CreateOrFail() error {
+	resp, err := v.HostClients.IAM.GetUser(&iam.GetUserInput{})
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	userArn := *resp.User.Arn
+	peerOwnerId := strings.Split(userArn, ":")[accountIDPosition]
```

Comment:
I think it would be good to have a helper function and basic tests for getting the owner ID.

Reply:
I was cheeky, nicked this from https://github.com/giantswarm/aws-operator/blob/master/client/aws/aws.go#L65

Considering time pressure, I'll break this out into a boyscouting issue. See https://github.com/giantswarm/aws-operator/issues/391